deploy/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:16-alpine AS BASE
```

Comment:
I am missing a variable that would connect to API of the server. We need to specify it in runtime i.e. in ENV variable, so e.g. 
```
ENV API_URL=http://localhost:8080/
```